error/src/util.rs

```diff
@@ -23,7 +24,27 @@ macro_rules! assert_error_eq {
     }
 }
 
-/// TODO(doc): @keroro520
+/// A macro to implement conversion from source type to target type with an implicit error kind.
+///
+/// ## Examples
+///
+/// ```rust
+/// impl_error_conversion_with_kind!(SourceType, error_kind, TargetType)
```

Comment:
```suggestion
/// ```ignore
/// impl_error_conversion_with_kind!(SourceType, error_kind, TargetType);
```

I didn't run the tests, but I guess this code couldn't pass `cargo run`.